package.json

```diff
@@ -38,38 +41,38 @@
     "argv-parse": "^1.0.1",
     "babel-eslint": "^10.0.0",
     "babel-loader": "^8.0.0",
-    "compression-webpack-plugin": "~1.1.11",
     "cross-env": "^5.2.0",
-    "css-loader": "^0.23.1",
+    "css-loader": "4.3.0",
```

Comment:
Is this an update we can do independent of the webpack change?

Reply:
`css-loader` 4 requires webpack 4, so we can't do it separetly